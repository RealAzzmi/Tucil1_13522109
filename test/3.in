9
6 6
7A 55 E9 1C BD 55 
BD 1C BD 1C E9 BD 
55 1C 1C 1C 55 55 
1C 1C 55 7A BD 55 
E9 1C 1C 1C BD E9 
55 E9 7A 55 E9 E9 
200
55 55 7A 
8
BD E9 E9 
-9
E9 1C E9 
-16
BD 7A 7A 
0
7A BD 
-20
55 1C BD 
-30
BD 55 E9 
-19
55 7A E9 
5
55 7A 
0
E9 1C 
-31
1C 55 
-44
7A 7A 
-11
7A 1C 
-5
1C BD 55 
-5
E9 1C 
10
E9 55 
-46
7A 1C 
-28
E9 BD 7A 
-13
BD E9 
-7
7A 1C 
-12
7A 1C 55 
-22
55 BD 
5
E9 7A 
-40
1C E9 BD 
-42
BD 55 
-17
55 E9 7A 
-26
55 55 BD 
5
1C E9 
-17
1C 1C 1C 
-29
7A E9 
-29
55 55 
-25
BD 55 BD 
-7
1C 1C 
-12
1C E9 BD 
-29
E9 E9 7A 
-11
7A BD 
-17
55 7A 7A 
-37
BD 1C 
-6
1C BD 
-1
1C BD BD 
-3
BD 1C 7A 
-40
7A BD BD 
-22
7A 1C 1C 
-10
BD 1C 
-28
1C 1C BD 
-34
E9 1C 
-2
E9 E9 E9 
-33
1C BD 
-14
E9 1C 
-3
7A BD 
-15
E9 55 
-1
E9 7A 
-29
BD 1C BD 
-50
BD 1C 
-31
E9 E9 
-39
7A E9 E9 
-35
BD 1C 
-9
E9 E9 
-3
55 7A 1C 
5
BD 1C 
-35
BD 1C 
-8
E9 1C 
-6
1C 7A 1C 
-15
1C E9 
2
1C 55 
-28
55 1C 55 
8
BD 7A 
8
E9 E9 
-40
E9 7A 
-50
E9 7A 7A 
-39
7A 7A 
-36
55 BD BD 
-14
7A 55 E9 
-37
1C E9 BD 
-28
7A E9 1C 
-43
BD 55 
-27
7A 7A E9 
-18
E9 7A 
-3
E9 BD 
-32
E9 E9 
-9
BD 55 1C 
-8
BD BD 
-31
7A 1C 
2
55 55 7A 
-18
55 1C 
-30
1C 55 
-41
BD E9 1C 
-12
1C 1C 
-22
BD BD 55 
-42
55 1C BD 
-30
55 55 7A 
6
BD BD 
-11
1C BD 
-33
55 1C 55 
-1
7A 7A 
9
55 E9 
-3
BD E9 
-9
7A 55 
3
E9 1C 
-37
1C 55 
-11
55 55 
-39
55 55 1C 
-43
55 1C 
1
BD 7A 
4
E9 7A 
-28
E9 1C E9 
-42
1C 7A 7A 
1
55 E9 
-38
1C 55 E9 
-20
E9 1C 
-32
7A E9 
-20
E9 1C 
-23
7A BD 1C 
2
7A 1C 
-45
BD 1C 
-7
E9 1C 
-35
1C E9 
6
7A 1C 
-12
7A 1C BD 
-15
7A E9 55 
-12
55 55 
-35
7A BD 55 
-15
BD E9 
8
BD 1C 7A 
-33
55 55 
-37
BD 7A 1C 
6
E9 E9 
-16
7A E9 
9
E9 1C BD 
-10
7A 7A E9 
9
7A 55 
-6
55 55 1C 
-19
1C E9 
-13
E9 1C 
-3
1C 7A E9 
-19
E9 BD E9 
-20
BD E9 7A 
-15
55 7A BD 
-4
55 BD E9 
5
E9 BD 
-17
E9 7A 7A 
9
7A 1C 
-44
BD E9 7A 
-15
55 7A BD 
-48
BD 55 1C 
-5
1C BD 
-45
55 7A 
4
55 BD BD 
9
7A 55 1C 
0
1C BD 
-7
1C 7A 55 
0
55 E9 E9 
-1
1C 7A 
-18
E9 BD E9 
7
55 E9 
-33
55 7A 
-27
55 BD 
-17
E9 55 7A 
-33
7A 7A 1C 
-25
E9 7A E9 
-7
1C BD 
-11
55 E9 
-13
7A E9 
-44
1C 1C BD 
5
BD E9 55 
7
7A 1C 
-9
55 7A 55 
-6
7A 55 55 
-16
E9 BD 7A 
-39
55 7A 1C 
-40
1C 55 E9 
-16
7A 7A 55 
6
7A BD 
7
BD E9 55 
-34
55 55 E9 
7
BD BD BD 
5
BD 7A 
-10
1C 7A 55 
-32
55 1C 
-8
BD 7A 
-10
7A 55 7A 
-36
55 7A 
2
55 55 BD 
5
55 1C BD 
-7
1C E9 
-37
E9 BD 7A 
-38
E9 E9 
-50
55 BD 
-45
55 55 E9 
-28
7A BD 
4
E9 E9 
-43
E9 1C 55 
-17
BD BD 
-27
1C 55 
-41
E9 E9 
1
BD 7A E9 
-42
BD E9 
-19
7A 55 
-3
E9 55 1C 
9
E9 1C 55 
-5