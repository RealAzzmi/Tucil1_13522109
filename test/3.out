Tidak ditemukan solusi yang menghasilkan total hadiah yang positif.
1727.25 ms